src/plugins/koops_event.conf

```diff
@@ -30,23 +30,23 @@ EVENT=post-create type=Kerneloops remote!=1
 #EVENT=report_Kerneloops type=Kerneloops
         reporter-kerneloops
 
-EVENT=report_Bugzilla type=Kerneloops
+EVENT=report_Bugzilla type=Kerneloops mce!=non-fatal mce!=fatal
         reporter-bugzilla -b \
                 -F /etc/libreport/plugins/bugzilla_format_kernel.conf
 
 # Send micro report
-EVENT=report_uReport type=Kerneloops
+EVENT=report_uReport type=Kerneloops mce!=non-fatal mce!=fatal
```

Comment:
Thank you for next insight.
I tried to report using notify event, it respects new settings and doesn't send uReport

Regarding error logs, I think it is ok, this is all I get:
Mar 22 17:21:56 localhost.localdomain abrt-dump-journal-oops[866]: abrt-dump-journal-oops: Found oopses: 1                                                                                                         
Mar 22 17:21:56 localhost.localdomain abrt-dump-journal-oops[866]: abrt-dump-journal-oops: Creating problem directories
Mar 22 17:21:57 localhost.localdomain abrt-dump-journal-oops[866]: Reported 1 kernel oopses to Abrt
Mar 22 17:21:59 localhost.localdomain abrt-server[1068]: Cannot notify '/var/spool/abrt/oops-2017-03-22-17:21:56-866-0' via uReport: No processing is specified for event 'report_uReport'